### ISSUE TITLE
Gitrob all the Things

### ISSUE THREAD

**Issue by myhndl** [polyswarm/development#1]
We should check ourselves shortly after we wreck ourselves.

I think we should add gitrob (https://github.com/michenriksen/gitrob) to the CI processes of most (all?) of our repos.

**Comment by myhndl** [comment]
For now, we need this on:
1. [ ] contracts
1. [ ] polyswarmd
1. [ ] polyswarm-client
1. [ ] relay

We'll expand as we open source more things.

**Comment by myhndl** [comment]
@zv looked into this and it's really not geared for any sort of automation. It stands up an HTTP server and points to you look at a webpage of results. STDOUT is limited.

As such, I'd like to pivot on this - this is not part of CI, just part of a standard release process. Updating this ticket and moving to the appropriate column.

**Comment by telwell** [comment]
For posterity I've saved the `json` of my scan and attached it here. After creating the github key I ran `gitrob -no-expand-orgs polyswarm` to prevent it from scanning all of my personal repos. This file can be loaded later if anyone is curious, but I didn't find anything too interesting.

The only things of note were:
- There seem to be a lot of certificates in the archived `hive` repo such as [this](https://github.com/polyswarm/hive/blob/6434668f5bed618c7200147b41d5ec42f7caf28e/tls/certs/ca-chain.cert.pem). I don't think this is a problem but it was worth pointing out.
-  [This](https://github.com/polyswarm/sidechain/blob/3c0448102464a32fd709729c9da298da0603873a/sidechain/password.txt) initially looked like a password but is not sensitive. 
- Anything with the string `password` in the filename will be flagged. There were some `css` files which were flagged but contained no sensitive information.

TL;DR this scan was useful but didn't reveal anything concerning.

[poly_gitrob.json.gz](https://github.com/polyswarm/development/files/2715718/poly_gitrob.json.gz)
